tests/jerry/array-prototype-sort.js

```diff
@@ -43,9 +43,20 @@ function f(arg1, arg2) {
 	}
 }
 
-var sorted = array.sort(f);
+array.sort(f);
 for (i = 0; i < array.length; i++) {
-	assert(sorted[sorted.length - i - 1] === i);
+	assert(array[array.length - i - 1] === i);
+}
+
+// Sorting sparse array
+var array = [1,,2,,3,,4,undefined,5];
+var expected = [1,2,3,4,5,,,,];
```

Comment:
Yes, initially. But we copy values into a local array, so that we can sort them. When we do this both will become undefined, since we don't have a way to represent them differently.